static/app/views/releases/components/sideTable.tsx

```diff
@@ -0,0 +1,38 @@
+import styled from '@emotion/styled';
+
+import {space} from 'sentry/styles/space';
+
+type Props = {
+  children?: React.ReactNode;
+  type?: undefined | 'error' | 'warning';
+};
+
+export const Table = styled('dl')<{noMargin?: boolean}>`
+  ${p => (p.noMargin ? 'margin-bottom: 0;' : null)}
+`;
+
+export function TableRow({type, children}: Props) {
+  return <Row type={type}>{children}</Row>;
+}
+
+const Row = styled('div')<{type: Props['type']}>`
+  ${p => p.theme.overflowEllipsis};
+  font-size: ${p => p.theme.fontSizeMedium};
+  padding: ${space(0.5)} ${space(1)};
+  font-weight: normal;
+  line-height: inherit;
+
+  background-color: ${p => {
+    switch (p.type) {
+      case 'error':
+        return p.theme.red100 + ' !important';
+      case 'warning':
+        return 'var(--background-warning-default, rgba(245, 176, 0, 0.09)) !important';
+      default:
+        return 'inherit';
+    }
+  }};
```

Comment:
when you're doing things with background colors you'll need to check that dark mode still looks okay

Reply:
nevermind this is the same as KeyValueTable but in another format

Reply:
Ah - yea, these were copied from `KeyValuetable` so i assume they're all fine 🤷